xml/System.Data.SqlClient/SqlConnectionStringBuilder.xml

```diff
@@ -2017,7 +2017,7 @@ False
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets or sets a Boolean value that indicates if security-sensitive information, such as the password, is not returned as part of the connection if the connection is open or has ever been in an open state.</summary>
+        <summary>Gets or sets a Boolean value indicating if security-sensitive information, such as the password or access token, should be returned as part of the connection string on a connection created with this <see cref="T:System.Data.SqlClient.SqlConnectionStringBuilder" /> after that connection has ever been in an open state.  This property should only be set to <see langword="true" /> if your application has a specific need to read the password out of an already-opened database connection.  The default value of <see langword="false" /> is the more secure setting; using <see langword="true" /> for this property opens your application to security risks such as accidentally logging or tracing the database password.</summary>
         <value>The value of the <see cref="P:System.Data.SqlClient.SqlConnectionStringBuilder.PersistSecurityInfo" /> property, or <see langword="false" /> if none has been supplied.</value>
         <remarks>
           <format type="text/markdown"><![CDATA[
```

Comment:
Then I would put this info in the markdown Remarks section (I'd make a suggestion, but GitHub isn't letting me):

> [!NOTE]
> This property should only be set to `true` if your application has a specific need to read the password out of an already-opened database connection. The default value of `false` is the more secure setting. Using `true` for this property opens your application to security risks such as accidentally logging or tracing the database password.